index.js

```diff
@@ -1,13 +1,37 @@
+// Load environment variables from .env file
+require('dotenv').config();
+
 const express = require('express');
 const path = require('path');
+const mongoose = require('mongoose');
+const auth = require('./lib/routes/auth');
 
 const port = process.env.PORT || 8080;
 const app = express();
 
+app.use('api/auth', auth);
```

Comment:
be careful here. 
the `/` is missing and would be an issue if you move files around.

```suggestion
app.use('/api/auth', auth);
```